build/cluster-operator-ansible/playbooks/cluster-api-prep/deploy-cluster-api.yaml

```diff
@@ -92,8 +92,25 @@
       src: files/cluster-api-template.yaml
       dest: "{{ temp_file.path }}"
 
+  - name: set image names
+    set_fact:
+      l_cluster_api_image: "{{ cluster_api_image|default('registry.svc.ci.openshift.org/openshift-cluster-operator/kubernetes-cluster-api:latest', true) }}"
+      l_cluster_api_image_pull_policy: "{{ cluster_api_image_pull_policy|default('Always', true) }}"
+      l_machine_controller_image: "{{ machine_controller_image|default('registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator:latest', true) }}"
+      l_machine_controller_image_pull_policy: "{{ machine_controller_image_pull_policy|default('Always', true) }}"
+
   - name: process template
-    shell: "oc process -f {{ temp_file.path }} -p CLUSTER_API_NAMESPACE={{ cluster_api_namespace }} -p SERVING_CA={{ cluster_api_ca_bundle }} -p SERVING_CERT={{ cluster_api_cert }} -p SERVING_KEY={{ cluster_api_key }} -p BOOTSTRAP_KUBECONFIG='{{ bootstrap_kubeconfig.content }}' | oc apply -f -"
+    shell: |-
+      oc process -f {{ temp_file.path }} \
+         -p CLUSTER_API_NAMESPACE={{ cluster_api_namespace }} \
+         -p SERVING_CA={{ cluster_api_ca_bundle }} \
+         -p SERVING_CERT={{ cluster_api_cert }} \
+         -p SERVING_KEY={{ cluster_api_key }} \
+         -p BOOTSTRAP_KUBECONFIG='{{ bootstrap_kubeconfig.content }}' \
+         -p IMAGE='{{ l_cluster_api_image }}' \
```

Comment:
Is there a way that these default image can be written in one places. As it is, there are 4 places where the default image is defined. I fear that the defaults will diverge.

Reply:
We really only need the default in the initial deployment playbook. I'll remove it from elsewhere.